chore(helm): AS-675 Standardize and configure probes (#384)
voxel51/fiftyone-teams-app-deploy: helm/docs/upgrading.md

   fiftyone migrate --info
   ```

#### FiftyOne Enterprise v2.9+ Startup Probe Changes

<!-- Differs from docker-compose docs -->

> FiftyOne Enterprise moves the `<settings>.service.startup` values in
> the `values.yaml` to the `<settings>.startup` section.

Reviewer: ```suggestion
In `v2.9.0`, the `values.yaml` location of startup probe settings
moved from `<settings>.service.startup` to `<settings>.startup` section.
```

Author: See https://github.com/voxel51/fiftyone-teams-app-deploy/pull/384/commits/54c68a184cee027db3c5605306c6a135fcb0d565

> For example, the following v2.8.2 `values.yaml`

Reviewer: ```suggestion
If your `values.yaml` contains
```

Author: See https://github.com/voxel51/fiftyone-teams-app-deploy/pull/384/commits/54c68a184cee027db3c5605306c6a135fcb0d565

```yaml
appSettings:
  service:
    startup:
      failureThreshold: 10
      periodSeconds: 15
```

> would be converted to the following v2.9.0 `values.yaml`

Reviewer: ```suggestion
move `appSettings.service.startup` to `appSettings.startup`
```

Author: See https://github.com/voxel51/fiftyone-teams-app-deploy/pull/384/commits/54c68a184cee027db3c5605306c6a135fcb0d565

```yaml
appSettings:
  startup:
    failureThreshold: 10
    periodSeconds: 15
```

> 

Reviewer: ```suggestion

Repeat for each occurrence of `<settings>.service.startup` to `<settings>.startup`.
```

Author: See https://github.com/voxel51/fiftyone-teams-app-deploy/pull/384/commits/54c68a184cee027db3c5605306c6a135fcb0d565
